prac_05/hex_colours.py

```diff
@@ -0,0 +1,28 @@
+"""
+CP1404/CP5632 Practical
+Hex Colours Dictionary Lookup
+"""
+
+colour_codes = {
+        'blue': '#0018a8',
+        'cyan': '#00ffff',
+        'crimson': '#dc143c',
+        'daffodil': '#ffff31',
+        'ebony': '555d50',
+        'emerald': '#50c878',
+        'firebrick': '#ff3030',
+        'gray': '#bebebe',
+        'heliotrope': '#df73ff',
+        'magenta': '#ff00ff'
+        }
+
+while True:
+    try:
+        colour_choice = input('Select your colour:').lower()
+        if colour_choice == '':
+            break
+        print(f'The code for {colour_choice} is {colour_codes[colour_choice]}')
+        break
```

Comment:
How come you used single quote marks for this print statement, but used double quote marks for the last print statement?

Also I feel like the break here will cause the loop to only allow the user to input one colour...